src/components/uielements/assets/assetInput/assetInput.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+import { storiesOf } from '@storybook/react'
+import { tokenAmount } from '@thorchain/asgardex-token'
+
+import AssetInput from './assetInput'
+
+storiesOf('Components/Assets/AssetInput', module).add('default', () => {
+  return (
+    <div style={{ padding: '20px' }}>
+      <AssetInput
+        title="swap amount"
+        status="slip 2%"
+        amount={tokenAmount(12345).amount()}
```

Comment:
[hint] You can use `bn` from `@thorchain/asgardex-util` to create a `BigNumber` more easily: 
```
bn(12345)
```